src/terminal/message.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
```

Comment:
normally would be against this type of hammer, but I think this is an appropriate nail.

Reply:
i'm confused is it a hammer or a nail 😉